CHANGES

```diff
@@ -3,6 +3,10 @@ Not released
 - Add family search
```

Comment:
ok this changes is old.. the family search is already released, so you can remove it..
another thing is can you add also the update to README.md?